packages/sdk/src/spaceWithEntitlements.test.ts

```diff
@@ -3,162 +3,45 @@
  */
 
 import {
-    getDynamicPricingModule,
-    everyoneMembershipStruct,
-    waitFor,
+    createTownWithRequirements,
     createUserStreamAndSyncClient,
-    createSpaceAndDefaultChannel,
+    customCheckOp,
+    erc20CheckOp,
+    everyoneMembershipStruct,
     expectUserCanJoin,
-    setupWalletsAndContexts,
-    linkWallets,
+    expectUserCannotJoinSpace,
     getNftRuleData,
-    twoNftRuleData,
     getXchainSupportedRpcUrlsForTesting,
-    erc20CheckOp,
-    customCheckOp,
+    linkWallets,
     nativeCoinBalanceCheckOp,
     oneEth,
-    twoEth,
-    threeEth,
     oneHalfEth,
+    setupWalletsAndContexts,
+    threeEth,
+    twoEth,
+    twoNftRuleData,
+    waitFor,
 } from './util.test'
 import { dlog } from '@river-build/dlog'
 import { MembershipOp } from '@river-build/proto'
-import { Client } from './client'
 import {
     Address,
     CheckOperationType,
-    ETH_ADDRESS,
     LogicalOperationType,
-    MembershipStruct,
     NoopRuleData,
     Operation,
     OperationType,
-    Permission,
     TestCustomEntitlement,
     TestERC20,
     TestERC721,
     TestEthBalance,
     treeToRuleData,
-    IRuleEntitlementV2Base,
-    ISpaceDapp,
-    encodeRuleDataV2,
     encodeThresholdParams,
     createExternalNFTStruct,
 } from '@river-build/web3'
 
 const log = dlog('csb:test:spaceWithEntitlements')
 
-// Users need to be mapped from 'alice', 'bob', etc to their wallet addresses,
-// because the wallets are created within this helper method.
-async function createTownWithRequirements(requirements: {
```

Comment:
This method is now shared with the spaceWithV2Entitlements test suite and was moved to util.test.ts.